packages/wonder-blocks-i18n/package.json

```diff
@@ -17,6 +17,9 @@
   "dependencies": {
     "@babel/runtime": "^7.16.3"
   },
+  "peerDependencies": {
+    "react": "16.14.0"
```

Comment:
Do we need to do this for other WB packages too?

Reply:
It's set on the rest of the packages.